napari/components/_layers_list/model.py

```diff
@@ -67,14 +67,13 @@ def __init__(self, viewer=None):
                          lookup={str: lambda q, e: q == e.name})
         self._viewer = None
 
-        self.changed.added.connect(_add)
```

Comment:
believe it or but there was a reason for this ordering and what i think may be one of the last bugs has now been fixed because of this change!